MergeCalendarsTogether.gs

```diff
@@ -11,17 +11,59 @@ const SYNC_DAYS_IN_FUTURE = 30;
 // Set this to "false" when your happy with the debug output!
 const DEBUG_ONLY = true;
 
+// Configure event summaries to ignore. These values are used with
+// RegExp.test() so when just a string literal, they act like a case-sensitive
+// "contains" check. If you want more control, use the line start (^) and/or
+// line end ($) regex symbols.
+const IGNORE_LIST_REGEXES = [
+  // 'Contains Match',
+  // '^Starts With Match',
+  // 'Ends With Match$',
+  // '^Some Exact Match$',
+  // '^Exact start.*Exact end$', // with anything in the middle
+]
+
+// should we copy event descriptions?
+const USER_INCLUDE_DESC = true;
```

Comment:
Honestly, let's default this to "false" - After running this a while I've realized there's no reason I'd view the "clone" for details.
```suggestion
const USER_INCLUDE_DESC = false;
```

Reply:
Done :ok_hand: